### DISCUSSION TITLE
Writers workshop May 7

### DISCUSSION THREAD

**Discussion by tonyfast** [Quansight/writers-workshop#3] [New class]
# the goal of the workshop

the goal of the workshop is to write, not be right. 

this week we'll think about all the ways we create literature when working in open source. 

## teacher's preferred ground rules

* How to use the raise handle feature? Keeping the professor on track..
* I encourage y'all to interrupt me when I'm talking or demo'ing. 
  Sometimes I get distracted by my own special interests and this class if for you all.
* To help take notes, please respond to this discussion item and continue editing your one reply during class. i can see the changes on the discussion board as they happen.

## [call to action] :+1: :-1:  :smile: :confused:  :rocket:  :heart:  :eyes: :tada: attendance

react to this discussion so we can take attendance.

![image](https://user-images.githubusercontent.com/4236275/117461813-fd163180-af1b-11eb-901b-bfdee65d8bdc.png)

### Q1 how's your energy

* :+1: i feel energetic from this week
* :-1: i feel exhausted from this week

### Q2 how was the assignment

* :smile: the writing assignment felt good to complete
* :confused: the writing assignment was difficult
* :rocket: i was too busy writing other stuff this week

### Q3 how do you want to participate

* :heart:   i'm happy to be at the writers workshop this week
* :eyes: i want to just watch and listen
* :tada: i am okay being called on for discussions

## agenda

0. [:00-:07] chit chat
1. [:07-:12] Recap and introduce new folks

    Practice raising our hand.
    
    [Preview of our class docs!](https://quansight-writers-workshop.readthedocs.io/en/latest/qww/short%20programs.html)

2. [:12-:40] discuss the homework _and_ [submit an idea you'd like to read about](https://github.com/Quansight/writers-workshop/issues/new?assignees=&labels=idea%2C&template=suggestion.md&title=%F0%9F%92%A1+i+want+to+read+about+...).
3. [:40-:50] review some ideas submitted

  ## assignment

4. [:50-:60] wrapup

    edit and improve your story in the third person, separate yourself from your work.

**Comment by tonyfast** [comment]
These are my notes

**Comment by isabela-pf** [comment]
This repo is new! Welcome!
- We will take notes in discussions (like this) per meeting.
- The goal of this workshop is just to write. Don't judge yourself just on your own, remember to look at the content we are all making together.
- We have issue templates for what you'd like to read about. ❗Please fill one out today.

How Tony is publishing
- [Jupyter Book](https://jupyterbook.org/intro.html#contribute-to-jupyter-book)
- This makes it easy, but also we need to better accommodate non-jovyans (jupyter nerds) here.
- As long as there is content, we can figure out how to publish it. 

Homework
- Continue with the short story! Figure out how you would prefer to read it and make edits based on this perspective.

**Comment by MarsBarLee** [comment]
Notes:
- For sharing notebooks, sharing on NBViewer > Github, since with NBViewer, you can add CSS, JS
- If we want to edit our old notebooks, we can edit our own submodule. For example, this is [mine](https://github.com/MarsBarLee/writers-workshop/tree/8e021cb1422d741e3d0952e6f919d490748afe58).
- Possible bottlenecks: non-computational environments not suitable for our writing? 
- Currently in a 'shared kernel' = a variables Tony defines, now everyone can see
- My 'I want to read about...' [submission](https://github.com/Quansight/writers-workshop/issues/5)

**Comment by bjuncek** [comment]
My notes? 
- I don't actually know how to use any of the IPYNB stuff and feel like a n00b
-  interesting note: use `ipywidgets` as a thing
